ui-kit/src/main/kotlin/org/p2p/uikit/components/edittext/v2/NewUiKitEditText.kt

```diff
@@ -0,0 +1,48 @@
+package org.p2p.uikit.components.edittext.v2
+
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.res.use
+import android.content.Context
+import android.os.Parcelable
+import android.util.AttributeSet
+import org.p2p.uikit.R
+import org.p2p.uikit.components.edittext.UiKitEditTextSavedState
+import org.p2p.uikit.databinding.WidgetUiKitEditTextNewBinding
+import org.p2p.uikit.utils.focusAndShowKeyboard
+import org.p2p.uikit.utils.inflateViewBinding
+
+class NewUiKitEditText @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+) : ConstraintLayout(context, attrs) {
+    private val binding: WidgetUiKitEditTextNewBinding by lazy(LazyThreadSafetyMode.NONE, ::inflateViewBinding)
+
+    val values: NewUiKitEditTextValues = NewUiKitEditTextValues(binding)
```

Comment:
the only problem with this way of extracting code is that you violate the hollywood principle (https://deviq.com/principles/hollywood-principle). 

to get any value you need to call `binding.newEditText.values.inputText` 
to set listener or error `binding.newEditText.mutate().setError(...)`

I didn't want to make this view group big, that's why i separated set/get logic into a classes